apps/teamsjs-cc-app/tabs/src/helpers/writetoexcel.ts

```diff
@@ -0,0 +1,26 @@
+import { utils, write, writeFile } from "xlsx";
+
+import { ICapabilityStatus } from "../components/Tab";
+
+/**
+ * Creates a .csv file
+ * @param defaultRowList, type of ICapabilityStatus[]
+ * @param client, type of string 
+ */
+export function createCsv(defaultRowList: ICapabilityStatus[], client: string) {
```

Comment:
Should client really be a free-form string? Or can the type limit it to a set of valid values, possibly using an enumeration?